### DISCUSSION TITLE
Stabilizer and Kalman Estimator warnings

### DISCUSSION THREAD

**Discussion by EJKT** [bitcraze/discussions#207] [Q&A]
Hi, 
When running the propeller test and the battery test in the cfclient, I get the following warnings about the stabilizer and the Kalman estimator. My CF also sometimes just flips over mid-flight (I posted something about this before and we concluded it had something to do with too little time between controls). Still, in this instance, I was using the motion commander class and leaving a 0.5-second delay between simple blocking functions, and this flip-over happened again. I am wondering if it has something to do with the warnings in the cfclient (see image below). If not I would really like a solution or explanation because I am afraid that one of these times the crazyflie will be damaged in one of these falls. 
![image](https://user-images.githubusercontent.com/87431056/185582432-51473bb2-e81a-4fec-a9a7-151c140db05f.png)

It now also flips over after a few seconds, when simple commands like 'take-off' are given in the cfclient flight commander tab. 

Thanks in advance and please let me know ASAP.

**Comment by knmcguire** [comment]
We need more information from your system. The guidelines of this platform has some instructions on point 2 how to provide more debugging information: https://github.com/orgs/bitcraze/discussions/55,

**Reply by EJKT** [reply]
In fact, I did some inspection and my problem was simply due to the fact that the propellers weren’t attached well to the motors. I have noticed that putting them on partially as recommended in the tutorials is slightly risky since it poses the risk that you may experience what I did. I just pushed the propellers all the way down onto the motor axle and my problems are fixed. Never the less, I still have the console showing me the above warnings of which I posted a screenshot of the console output.

**Comment by krichardsson** [comment]
You get the warnings for the stabilizer loop and kalman estimator because the propeller test is using the CPU a bit too much and you can safely ignore them. The propeller test is not written in such a way that it plays nicely with the rest of the system since it is just testing the propellers on the ground. 

The flipping is not related to this (if you are using the standard firmware). Please start a new discussion for your flipping problem (with a proper title) and please let us know what firmware you are using, your system setup (positioning system and so on) and what python script you are running.
